controllers/storagecluster/job_templates_test.go

```diff
@@ -0,0 +1,135 @@
+package storagecluster
+
+import (
+	"context"
+	"testing"
+
+	openshiftv1 "github.com/openshift/api/template/v1"
+	api "github.com/openshift/ocs-operator/api/v1"
+	"github.com/stretchr/testify/assert"
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var expectedOcsRemovalParameterName = "FAILED_OSD_IDS"
+var expectedOcsExtendParameterName = "RECONFIGURE"
+
+var extendClusterCommandTest = []string{
+	"bash",
+	"-c",
+	`full_ratio_value=$(ceph osd dump | grep -m1 full_ratio | awk '{print $2}')
+	  echo $full_ratio_value
+	if [[ $full_ratio_value == "0.85"  ]]; then
+		   ceph osd set-full-ratio 0.87
+	else
+		   ceph osd set-full-ratio 0.85
+	fi
+	`,
+}
+var osdCleanupArgsTest = []string{
+	"ceph",
+	"osd",
+	"remove",
+	"--osd-ids=${FAILED_OSD_IDS}",
+}
```

Comment:
Defining these variables here will lead to problems if they ever get updated in the runtime code. I would suggest defining them in the code and referencing them in both places they're used.

Reply:
You mean defining inside the method where I'm using it? @jarrpa

Reply:
Correct, I suggest not doing that.